website/src/pages/Micro/Micro.jsx

```diff
@@ -0,0 +1,181 @@
+        navigate(`?ltag=${gitRefLeft}&rtag=${gitRefRight}`)
+    }, [gitRefLeft, gitRefRight]) 
+
+
+    return (
+        <div className='micro'>
+            <div className='micro__top justify--content'>
+                <div className='micro__top__text'>
+                    <h2>MicroBenchmark</h2>
+                    <span>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer a augue mi.
+                            Etiam sed imperdiet ligula, vel elementum velit.
+                            Phasellus sodales felis eu condimentum convallis.
+                            Suspendisse sodales malesuada iaculis. Mauris molestie placerat ex non malesuada.
+                            Curabitur eget sagittis eros. Aliquam aliquam sem non tincidunt volutpat. 
+                    </span>
+                </div>
+                <figure className='microStats'></figure>
+            </div>
+            <figure className='line'></figure>
+            <div className='micro__bottom'>
+                <div className='micro__bottom__title justify--content'>
+                    <h3>Comparing result for</h3>
+                    <figure className='micro__bottom__DropDownLeft flex--column' style={{ maxHeight: `${openDropDownLeft}px` }}>
+                        <span className='DropDown__BaseSpan'  onClick={() => openDropDown(openDropDownLeft, setOpenDropDownLeft)}>{gitRefLeft} <i className="fa-solid fa-circle-arrow-down"></i></span>
+                        {dataRefs.map((ref, index) => {
+                            return (
+                                <React.Fragment key={index}>
+                                    <figure className='dropDown--line'></figure>
+                                    <span className='dropDown__ref' onClick={() => {valueDropDown(ref, setGitRefLeft, setCommitHashLeft, setOpenDropDownLeft)}}>{ref.Name}</span>
+                                </React.Fragment>
+                            )
+                        })}
+                    </figure>
+                    {/* <h3>and</h3> */}
+                    <figure className='micro__bottom__DropDownRight flex--column' style={{ maxHeight: `${openDropDownRight}px` }}>
+                        <span className='DropDown__BaseSpan'  onClick={() => openDropDown(openDropDownRight, setOpenDropDownRight)}>{gitRefRight} <i className="fa-solid fa-circle-arrow-down"></i></span>
+                        {dataRefs.map((ref, index) => {
+                            return (
+                                <React.Fragment key={index}>
+                                    <figure className='dropDown--line'></figure>
+                                    <span className='dropDown__ref' onClick={() => valueDropDown(ref, setGitRefRight, setCommitHashRight, setOpenDropDownRight)}>{ref.Name}</span>
+                                </React.Fragment>
+                            )
+                        })}
+                    </figure>
```

Comment:
The comparison dropdown and text are different between the macro and micro comparison pages. I think it looks very nice on the macrobenchmark page, see the first screenshot. You can see the current style of the microbenchmark page on the second screenshot. What would it take to duplicate the same style and have a similar dropdown / text in micro?

**Macro:**
![image](https://github.com/vitessio/arewefastyet/assets/35779988/86832a3b-83cb-415d-9781-24e9bb0f04d7)

**Micro:**
![image](https://github.com/vitessio/arewefastyet/assets/35779988/faa48b60-4d0f-4ca7-8602-f19a6ee8dc20)

Reply:
I want to find a new design for both to have a dropdown directly integrated into the title. I'm still thinking about it, and I will come back to it once I have found a solution.

Reply:
Sounds good! Can you create an issue to track this down @Camillemtd?

Reply:
https://github.com/vitessio/arewefastyet/issues/389